src/admin/content-block/content-block.const.ts

```diff
@@ -34,33 +35,38 @@ import {
 	IFRAME_BLOCK_CONFIG,
 	IMAGE_BLOCK_CONFIG,
 	IMAGE_GRID_BLOCK_CONFIG,
-	INITIAL_ACCORDIONS_BLOCK_COMPONENT_STATES,
 	INITIAL_ACCORDIONS_BLOCK_STATE,
-	INITIAL_BUTTONS_BLOCK_COMPONENT_STATES,
+	INITIAL_ACCORDIONS_COMPONENTS_STATE,
 	INITIAL_BUTTONS_BLOCK_STATE,
-	INITIAL_CTAS_BLOCK_COMPONENT_STATES,
+	INITIAL_BUTTONS_COMPONENTS_STATE,
```

Comment:
de STATES op het einde duidde aan dat dit een array was. Heb je die interface aangepast zodat nu alles een array bevat? Anders zou ik die STATES vs STATE benaming behouden.

related quotes from andry:
> vraagje, had jij niet ooit een PR opengezet waar je ervoor gezorgd had dat de components state (content-block configs) altijd een array was ipv een array of object?

Reply:
ja, ik heb dit overal gelijk getrokken omdat het hier gaat over de config property **components**.**state** en idd met het in gedachte houden dat alles 1 type (array) gaat worden